library/src/scripts/forms/Button.tsx

```diff
@@ -61,7 +61,7 @@ export const getDynamicClassFromButtonType = (type: ButtonTypes | undefined) =>
             case ButtonTypes.CUSTOM:
                 return classes.custom;
             default:
-                return classes.standard;
```

Comment:
Shouldn't it return at least class `button`? or whatever?

Reply:
I'm going to leave it as is. This function is to explicitly set a standard class. If you don't want a standard one, you're on your own.